nltk_contrib/timex3.py

```diff
@@ -81,29 +77,29 @@ def tag(text):
 # Hash function for week days to simplify the grounding task.
 # [Mon..Sun] -> [0..6]
 hashweekdays = {
-    'Monday': 0,
-    'Tuesday': 1,
-    'Wednesday': 2,
-    'Thursday': 3,
-    'Friday': 4,
-    'Saturday': 5,
-    'Sunday': 6}
```

Comment:
These capitalized day and month names were not working in every scenario for me.  By converting to lowercase for input text these issues were resolved.